replay-assets/replay_command_handlers.js

```diff
@@ -1725,12 +1725,133 @@ function getInternalFunctionLocationProp(cdpProperties) {
   return getInternalProp(cdpProperties, '[[FunctionLocation]]');
 }
 
+/**
+ * String utility for function parameter parsing.
+ */
+function hasSubstringAt(haystack, needle, i) {
```

Comment:
Yep, that was my original name, but I decided against it because its so long 😆
![image](https://github.com/replayio/chromium/assets/282899/3060cfed-689d-472b-8831-8f718ce426e6)